core/src/main/java/bio/terra/pearl/core/model/participant/Enrollee.java

```diff
@@ -44,5 +44,5 @@ public class Enrollee extends BaseEntity {
     @Builder.Default
     private List<KitRequestDto> kitRequests = new ArrayList<>();
     @Builder.Default
-    private List<EnrolleeRelation> relations = new ArrayList<>();
+    private List<EnrolleeRelationDto> relations = new ArrayList<>();
```

Comment:
it seems like there will be cases where we want to load enrollee relations and *not* load the profiles too.  I recognize this follows the pattern of having a list of  KitRequestDto off of Enrollee, but TBH I didn't like the pattern there either.  

Let's keep this as `List<EnrolleeRelation> relations` and find another way to get the dtos you need to the frontend